### PR TITLE
Update

### PR DESCRIPTION
**Pull request by ReDeiPirati**
Here's the update:

- Fix typo (in core concept)
- Update the environment info
- Add Section (how to Stop Jupyter Jobs) in the `quick-start-jupyter` page

### CODE REVIEW

**Review by houqp — APPROVED** [review]

**Review by saiprashanths — APPROVED** [review]

File: docs/guides/environments.md [hunk 1]
```diff
@@ -5,12 +5,14 @@ these can be specified in the floyd [run](../commands/run.md) command using the
...
 
 If no `--env` is provided, it uses the `keras` image by default, which comes with Python
-3, Keras 2.0.4 and Tensorflow 1.1.0 pre-installed.
+3.6, Keras 2.1.3 and Tensorflow 1.5.0 pre-installed.
 
 | Framework | Env name (--env parameter)  |  Description              | Docker Image |
 | --------- | ------------------ | ------------------------ |-------------|
-| Keras | keras      | Tensorflow 1.1.0 + keras 2.0.6 on Python3.5. |  |
-|       | keras:py2  | Tensorflow 1.1.0 + keras 2.0.6 on Python2. |  |
+| Keras | keras      | Tensorflow 1.5.0 + keras 2.1.3 on Python3.6. |  |
```
Comment (saiprashanths) [inline docs/guides/environments.md]: Do we need to support a separate environment called Keras? Can this not fold under Tensorflow?

Comment (saiprashanths) [reply docs/guides/environments.md]: @houqp

**Review by saiprashanths — APPROVED** [review]

File: docs/guides/environments.md (hunk 1 above)
Comment (houqp) [reply docs/guides/environments.md]: yeah, we can fold it now. it wasn't the case when we created it.

Comment (saiprashanths) [reply docs/guides/environments.md]: Cool. @ReDeiPirati Can you please remove Keras env from the docs to simplify it?

Comment (ReDeiPirati) [reply docs/guides/environments.md]: Sure!

**Review by saiprashanths — APPROVED** [review]
Great, thanks!

**Pull request merged**